### PR TITLE
Update README.md #46

### PR DESCRIPTION
**Pull request by Olenka-Hryk**
Updated README.md: #46 
- Added badges for issues and pull requests
- Updated avatars for Development team
- Added icon for main tools

### CODE REVIEW

**Review by ohanovdmytro — APPROVED** [review]

**Review by docia — APPROVED** [review]

**Review by docia — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -251,40 +251,31 @@ Only the release should merge into the main branch!!!
...
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/css3/css3-original.svg" alt="CSS3" align="left" width="40" height="60"/>   
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/materialui/materialui-original.svg" alt="MaterialUI" align="left" width="40" height="60"/> 
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/javascript/javascript-original.svg" alt="JavaScript" align="left" width="40" height="60"/>   
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/react/react-original.svg" alt="React" align="left" width="40" height="60"/>  
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/redux/redux-original.svg" alt="Redux" align="left" width="40" height="60"/>
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/nodejs/nodejs-original.svg" alt="NodeJS" align="left" width="40" height="60"/>
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/mongodb/mongodb-original.svg" alt="MongoDB" align="left" width="40" height="60"/>   
+<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/eslint/eslint-original.svg" alt="Eslint" align="left" width="40" height="60"/> 
+<br />
+<br />
```
Comment (vlad-khrychov) [inline README.md:278]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -251,40 +251,31 @@ Only the release should merge into the main branch!!!
...
-[![@AntonOkun](https://avatars.githubusercontent.com/u/129941062?v=4)](https://github.com/AntonOkun)
-[![@Valent1n0o](https://avatars.githubusercontent.com/u/118978192?v=4)](https://github.com/Valent1n0o)
-
----
+[![@ohanovdmytro](https://avatars.githubusercontent.com/ohanovdmytro?s=130)](https://github.com/ohanovdmytro)
+[![@Olenka-Hryk](https://avatars.githubusercontent.com/Olenka-Hryk?s=130)](https://github.com/Olenka-Hryk)
+[![@Nazar-Lysak](https://avatars.githubusercontent.com/Nazar-Lysak?s=130)](https://github.com/Nazar-Lysak)
+[![@docia](https://avatars.githubusercontent.com/docia?s=130)](https://github.com/docia)
+[![@BodiaPutiak](https://avatars.githubusercontent.com/BodiaPutiak?s=130)](https://github.com/BodiaPutiak)
+<a href="https://github.com/m6xcom"><img src="https://avatars.githubusercontent.com/m6xcom" height="auto" width="130"></a>
```
Comment (vlad-khrychov) [inline README.md:259]:
```suggestion
[![@m6xcom](https://avatars.githubusercontent.com/m6xcom)](https://github.com/m6xcom)
```

File: README.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
-# Space2Study-node-BackEnd-mvp
+<h1 align="center"> Space2Study-node-BackEnd-mvp </h1>
```
Comment (vlad-khrychov) [inline README.md:1]:
```suggestion
# Space2Study-BackEnd
```

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Olenka-Hryk) [reply README.md]: The issue is resolved

File: README.md (hunk 3 above)
Comment (Olenka-Hryk) [reply README.md]: The issue is resolved

File: README.md (hunk 1 above)
Comment (Olenka-Hryk) [reply README.md:278]: Without <br /> content, the bottom content moves up to the top.

**Review by docia — APPROVED** [review]

**Review by Nazar-Lysak — APPROVED** [review]

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**